.travis.yml

```diff
@@ -6,6 +6,7 @@ php:
   - 7.1
   - 7.2
   - 7.3
+  - 7.4snapshot
```

Comment:
It seems per https://github.com/wpengine/phpcompat/pull/251#issuecomment-565703753 that Travis has the 7.4 image available now. I think that we should probably be using that before merge. Here's an example of folks using the new image already: https://github.com/google/site-kit-wp/pull/946/files